xapi-profiles-structure.md

```diff
@@ -452,12 +463,17 @@ Property | Type | Description | Required
 
 A Statement Template MUST NOT have both `objectStatementRefTemplate` and `objectActivityType`.
 
-The verb, object activity type, attachment usage types, and context activity types listed are called Determining Properties.
+The verb, object activity type, homepage, extensions, category context activity, attachment usage types, and context activity types listed are called Determining Properties.
 
-A Profile Author MUST change a Statement Template's `id` between versions if any of the Determining Properties, StatementRef properties, or rules change. Changes of `scopeNote` are not considered changes in rules.
+A Profile Author MUST change a Statement Template's `id` between versions if any of the Determining Properties, StatementRef properties, or rules change. 
+Changes of `scopeNote` are not considered changes in rules.
+A Profile Author SHOULD create a Statement Template with a single Determining Property of the Category Context Activity
+Other Statement Templates MAY use the Category Context Activity of the intended xAPI Profile in combination with another Determining Property. 
```

Comment:
I don't understand A Statement Template vs Other Statement Templates. Is this saying that at least one Statement Template in the profile must have the profile version reference determining property and then the rest do not have to?

Reply:
See comment above